index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TochukwuAndMarioPage</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media (max-width: 600px) {
+        .facet_sidebar {
+          display: none;
+        }
+      }
+      </style>
+  </head>
+  <body>
+    <nav id="navbar">
+      <header>
+        <div>
+         <a class="nav-link" href="#Introduction">Introduction</a>
+        </div>
+        <div>
+          <a class="nav-link" href="#What_you_should_already_know">What you should already know</a>
+        </div>
```

Comment:
I suggest you put shorter names with useful keywords to make your code much more understandable.